pybossa/api/__init__.py

```diff
@@ -469,25 +469,25 @@ def task_gold(project_id=None):
 @jsonpify
 @login_required
 @csrf.exempt
-@blueprint.route('/task/<task_id>/services/<service_name>/<service_version>', methods=['POST'])
+@blueprint.route('/task/<task_id>/services/<service_name>/<major_version>/<minor_version>', methods=['POST'])
```

Comment:
I was just curious about the ordering of the url route :), i.e., the placement of `task_id` and `service_name`.

Could you just give a little background on the ordering of the parameters? For example, `services/test-service-name` comes after `task/1` and I was just curious on this. Thank you! :)

`/api/task/1/services/test-service-name/1/31`

versus

(for example) `/api/services/test-service-name/1/31/task/1/`

Reply:
There is nothing special behind the ordering. Currently when a task is loaded the url looks like /project/<project_shortname>/task/<task_id>, since this new feature can only be used from a task that is already loaded an locked I decided to have the  /task/<task_id> before the service details "/service/<service_name>/<major_version>/<minor_version>".

The order of this /<service_name>/<major_version>/<minor_version> is the same format that is used by the service that we are calling.

Reply:
Thank you. :)